docs/API.md

```diff
@@ -94,6 +93,24 @@ interested in public data only can include a `ignore-auth=1` query so that only
 public information is returned. In this case, results requiring authorization
 (if any) will be omitted.
 
+### Channels
+
+Any entity in the charm store is considered to be part of one or more "channels"
+(think "distribution channels"). Currently supported channels are "unpublished",
+"development" and "stable". All entities are initially (and always) part
+of the "unpublished" channel; subsequent operations on the publish
```

Comment:
So if I upload a charm and then publish it, that charm is still in the "unpublished" channel?  Queries on the "unpublished" channel will return published charms?